src/modules/service-catalog/ServiceCatalogList.tsx

```diff
@@ -109,17 +127,20 @@ export function ServiceCatalogList({
             <Row wrap="wrap">
               {serviceCatalogItems.length !== 0 &&
                 serviceCatalogItems.map((record: ServiceCatalogItem) => (
-                  <StyledCol key={record.id} xs={12} sm={6} md={4} lg={3}>
+                  <Col key={record.id} xs={12} sm={6} md={4} lg={3}>
```

Comment:
Why are we removing the styled col with the margin bottom here?
It removes the space between items when there are multiple rows
<img width="1124" alt="Screenshot 2025-03-05 at 15 11 27" src="https://github.com/user-attachments/assets/02289c5c-17ce-48ca-9758-c27a24660ba3" />

Reply:
My bad. I will fix it.